chapters/annotations.tex

```diff
@@ -546,21 +570,23 @@ \subsection{Single Use of Class}\label{annotation-for-single-use-of-class}\label
 For state machines it is useful to have single instances of local classes.
 This can be done using:
 \begin{lstlisting}[language=modelica]
-annotation(singleInstance = true)
+/*literal*/ constant Boolean singleInstance;
```

Comment:
Let's make this the new representative of a component declaration style annotation where the default corresponds to a particular value:
```suggestion
/*literal*/ constant Boolean singleInstance = false;
```